pymc/tests/test_sampling.py

```diff
@@ -1606,7 +1606,7 @@ def test_linear_model(self):
             beta = pm.Normal("beta", 0, 0.1)
             mu = pm.Deterministic("mu", alpha + beta * x)
             sigma = pm.HalfNormal("sigma", 0.1)
-            obs = pm.Normal("obs", mu, sigma, observed=y)
+            obs = pm.Normal("obs", mu, sigma, observed=y, shape=x.shape)
```

Comment:
Not super familiar with shape inference. Why is it necessary to provide a `shape` argument here?

Reply:
This is part of the previous PR #6063